### PR TITLE
Add URLSearchParams and Console API

### PR DESCRIPTION
**Pull request by chicoxyzzy**
Actually URLSearchParams is part of URL spec. Should it be mentioned as separate global here?

### CODE REVIEW

**Comment by joyeecheung** [comment]
`URLSearchParams` and `URL` are both `Exposed=(Window,Worker)` in the URL spc, similar situations also happen to other Web API implementations in Node.js: e.g. the encoding spc exposes both `TextEncoder` and `TextDecoder`. Maybe it would be more reasonable to list the globals under different standards since we have these one-to-many relationships.

**Comment by chicoxyzzy** [comment]
Makes sense 👍 Moved `URLSearchParams` to `URL` API

File: GLOBALS.md [hunk 1]
```diff
@@ -38,3 +38,11 @@ Spec: https://www.w3.org/TR/wasm-js-api-1/
 Environments supported:
 - [Web](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly)
 - Node.js: not documented yet
+
+### Console API
+
+Spec: https://console.spec.whatwg.org
+
+Environments supported:
+- Web
```
Comment (chicoxyzzy) [inline GLOBALS.md]: MDN page doesn't exist yet. Do we have some other good documentation websites?

Comment (joyeecheung) [reply GLOBALS.md]: There is https://developer.mozilla.org/en-US/docs/Web/API/console

**Comment by littledan** [comment]
Thanks for this!

**Pull request merged**